sys/src/9/amd64/archamd64.c

```diff
@@ -182,29 +183,27 @@ cpuidhz(uint32_t *info0, uint32_t *info1)
 		}
 		DBG("cpuidhz: 0x2a: %#llx hz %lld\n", rdmsr(0x2a), hz);
 	}
-	else if(memcmp(&info0[1], "AuthcAMDenti", 12) == 0){
+	else if((memcmp(&info0[1], "AuthcAMDenti", 12) == 0) ||
```

Comment:
I detect confusion. 
The string is strange because of how they plan 9 cpuid works, not for any good reason. We can talk about it more but basically only ONE of these strings should be here.